nodes/color/combine_color.py

```diff
@@ -1,17 +1,94 @@
-import bpy, random
+import bpy, random, colorsys
```

Comment:
I know that comes from me. but looks like the random import isn't needed anymore